src/server/scan/listener/helpers/process-scan-complete.js

```diff
@@ -47,11 +47,13 @@ async function processScanComplete(server, uploadId, fileId) {
 }
 
 function isScanningComplete(files) {
-  return files.every((f) => isInfected(f.fileStatus) || f.delivered)
+  return files.every(
+    (file) => file.fileStatus === fileStatus.complete || file.delivered
```

Comment:
I think this `complete` check breaks the logic here - a file can be complete but not moved to the s3 bucket.

Reply:
This logs as intended

<img width="976" alt="Screenshot 2024-04-30 at 09 36 15" src="https://github.com/DEFRA/cdp-uploader/assets/2305016/ad6f0a49-766e-44ff-ad57-ad14b83f8ced">

Reply:
> a file can be complete but not moved to the s3 bucket.

Maybe this needs to be updated to `scanComplete`?